src/PiGui.h

```diff
@@ -0,0 +1,62 @@
+#include "RefCounted.h"
+#include "graphics/opengl/RendererGL.h"
+#include "Lua.h"
+#include "LuaRef.h"
+#include "LuaTable.h"
+#include "FileSystem.h"
+#include "imgui/imgui.h"
+#include "imgui/examples/sdl_opengl3_example/imgui_impl_sdl_gl3.h"
+
+/* Class to wrap ImGui. */
+class PiGui : public RefCounted {
+public:
+  static ImFont *pionillium12;
```

Comment:
Me thinks indentation in this file is 2 spaces, instead of one tab.

I don't know if you're using dtrtindent, like me, but it only works on files with some indentation already in place. When starting a new file, it will revert to what ever setting you have for your editor.